plugins/woocommerce/src/RestApi/Routes/V4/OrderNotes/Controller.php

```
@@ -0,0 +1,365 @@
+		 */
+		return apply_filters( $this->get_hook_prefix() . 'collection_params', $params, $this );
+	}
+
+	/**
+	 * Prepare links for the request.
+	 *
+	 * @param mixed           $item WordPress representation of the item.
+	 * @param WP_REST_Request $request Request object.
+	 * @return array
+	 */
+	protected function prepare_links( $item, WP_REST_Request $request ): array {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)